src/dafny/Results.dfy

```diff
@@ -0,0 +1,148 @@
+// RUN: %verify "%s"
+
+/*******************************************************************************
+ *  Copyright by the contributors to the Dafny Project
+ *  SPDX-License-Identifier: MIT 
+ *******************************************************************************/
+
+module {:options "--function-syntax:4"} Dafny.Results {
```

Comment:
For the record I still don't think this is an improvement over `Wrappers`, especially not enough to pay the change tax for anyone already using any of this code. But I'll allow this for now just because we're still allowing ourselves to make breaking changes in the future. :)

Reply:
Note they already have to change to a top-level Dafny module name.

Reply:
An example of how a not-great-but -temporary solution can become permanent